plugin_ide.core.tests/src-lang/melnorme/lang/ide/core/engine/StructureModelTest.java

```diff
@@ -60,8 +60,7 @@ public FixtureSourceModelManager() {
 			super(new DocumentReconcileManager(), new ProblemMarkerUpdater());
 		}
 		
-		public final SourceFileStructure DEFAULT_STRUCTURE = new SourceFileStructure(null, null, 
-			(Indexable<ParserError>) null);
+		public final SourceFileStructure DEFAULT_STRUCTURE = new SourceFileStructure(null, (Indexable<ParserError>) null);
```

Comment:
True. In one source file, the location is always the same. However, for the index, the `StructureElement`s can be different. There are several solutions for that problem. The one I committed seemed to be the most straight-forward to me. What do you think?